From 87cdb80f62200a911ed40cdd6a4d7fdfc740233f Mon Sep 17 00:00:00 2001
From: Dan Villiom Podlaski Christiansen <dchristiansen@zimbra.com>
Date: Mon, 13 Oct 2014 12:21:13 +0200
Subject: [PATCH] add grunt-cli to list of dependancies

This allows a single 'yarn install' to install everything required to
build TinyMCE.
---
diff -aur -x node_modules /tmp/root/tinymce-4.9.11/package.json tinymce-4.9.11/package.json
--- /tmp/root/tinymce-4.9.11/package.json	2020-07-13 04:14:19.000000000 +0000
+++ tinymce-4.9.11/package.json	2022-08-10 05:30:02.971158352 +0000
@@ -42,6 +42,7 @@
     "@ephox/wrap-jsverify": "2.0.1",
     "awesome-typescript-loader": "^5.2.0",
     "grunt": "~1.0.2",
+    "grunt-cli": "^1.2.0",
     "grunt-contrib-clean": "~1.1.0",
     "grunt-contrib-copy": "~1.0.0",
     "grunt-contrib-less": "~1.4.1",
